app/src/main/res/xml/account_settings_preferences.xml

```diff
@@ -1,54 +1,54 @@
-<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <androidx.preference.Preference
-        app:iconSpaceReserved="false"
+    <Preference
         android:key="@string/pref_key_sync_now"
-        android:title="@string/preferences_sync_now" />
+        android:title="@string/preferences_sync_now"
+        app:icon="@drawable/mozac_ic_sync"
+        android:layout="@layout/account_preference_description_icon" />
```

Comment:
It seems like reserving the icon space is sufficient here

Reply:
But if i just set app:icon here icon will stay white on theme changes because we can't set tint="?primaryText" in preference.
I've created that layout for this. If you know a way, i can use it.

Reply:
Generally we add tints using Kotlin for cases like this. 

In `AccountSettingsFragment`, you can add something like this:
```kt
        preferenceSyncNow.apply {
            ...

            icon = icon.mutate().apply { 
                setTint(context.getColorFromAttr(R.attr.primaryText))
            }
        }
```

Reply:
Great i'll use it.